python/pysrc/kaskada/sources/arrow.py

```diff
@@ -11,13 +11,22 @@
 import pyarrow.parquet
 
 from .source import Source
+from .source import TimeUnit
 
 
 class Pandas(Source):
     """Source reading data from Pandas dataframe."""
 
     def __init__(
-        self, dataframe: pd.DataFrame, *, schema: Optional[pa.Schema] = None, **kwargs
+        self,
+        dataframe: pd.DataFrame,
+        *,
+        time_column_name: str,
```

Comment:
`time_column_name` is long -- thoughts on `time_column` or `time_name`?

@kerinin @epinzur @jordanrfrazier